packages/react-jsx-highcharts/src/components/PlotBandLine/index.js

```diff
@@ -1,6 +1,7 @@
 import PlotBand from './PlotBand';
 import PlotLine from './PlotLine';
 import PlotBandLineLabel from './PlotBandLineLabel';
+import usePlotBandLine from './usePlotBandLine';
 
 const ChartPlotBand = PlotBand;
```

Comment:
I think we could drop the `Chart` prefixes on these variables in the index files - they made sense when they were the output of a `provideChart` HOC, but probably just a weird legacy artefact now.

Reply:
Done.

Reply:
It turned these consts are actually needed. Otherwise they were dropped from production build. I guess it has something to do with webpack tree shaking.

I reverted the change.